python/plugins/processing/gui/menus.py

```diff
@@ -67,11 +67,11 @@
 geoprocessingToolsMenu = vectorMenu + "/" + Processing.tr('&Geoprocessing Tools')
 defaultMenuEntries.update({'native:buffer': geoprocessingToolsMenu,
                            'native:convexhull': geoprocessingToolsMenu,
-                           'qgis:intersection': geoprocessingToolsMenu,
-                           'qgis:union': geoprocessingToolsMenu,
-                           'qgis:symmetricaldifference': geoprocessingToolsMenu,
+                           'native:intersection': geoprocessingToolsMenu,
+                           'native:union': geoprocessingToolsMenu,
+                           'native:symmetricaldifference': geoprocessingToolsMenu,
                            'native:clip': geoprocessingToolsMenu,
-                           'qgis:difference': geoprocessingToolsMenu,
+                           'native:difference': geoprocessingToolsMenu,
```

Comment:
I think we should start throwing a warning when a menu item is missing - we consistently hit this issue as algorithms are ported to c++

Reply:
@nyalldawson , yeah, that'd probably be useful on the short-term.